hoomd/mpcd/tune.py

```diff
@@ -10,10 +10,10 @@
 
 import hoomd
 from hoomd.mpcd import _mpcd
-from hoomd.operation import Tuner
+from hoomd.operation import TriggeredOperation
 
 
-class ParticleSorter(Tuner):
+class ParticleSorter(TriggeredOperation):
```

Comment:
When I was figuring out the base class for the fillers, I decided this should probably just be a `TriggeredOperation` (and not a tuner) so that this doesn't get put in the regular list of `tuners`.

Reply:
Agreed.